### ISSUE TITLE
Connection stop doesn't stop gracefully: Session destroy request failed

### ISSUE THREAD

**Issue by Germuxas** [mysteriumnetwork/node#822]
1. Download Windows release https://github.com/mysteriumnetwork/mysterium-vpn/releases/tag/0.8-rc1
2. Connect to some server e.g. Greece
3. Try to disconnect, and disconnect shows error about "Disconnect dailed"
4. Logs goes like this:
```
tail -f /c/Users/Mysterium/AppData/Roaming/MysteriumVPN/*.log

...
...
2019-03-26T12:26:36.3694141 [Info] [client-management] Shutdown
2019-03-26T12:26:36.3694141 [Info] [client-management] Shutdown finished
2019-03-26T12:26:36.3694141 [Info] [connection-manager] Cancelling connection initiation: context canceled
2019-03-26T12:26:36.3694141 [Debug] [session-stats-sender] stopping
2019-03-26T12:26:36.3704203 [Warn] [client-management] Connection failed to read. read tcp 127.0.0.1:1355->127.0.0.1:1356: wsarecv: An existing connection was forcibly closed by the remote host.
2019-03-26T12:26:36.3704203 [Info] [client-management] Event consumer is done
2019-03-26T12:26:36.3704203 [Warn] [client-management] Middleware stop error. write tcp 127.0.0.1:1355->127.0.0.1:1356: wsasend: An existing connection was forcibly closed by the remote host.
2019-03-26T12:26:36.3704203 [Warn] [client-management] Middleware stop error. write tcp 127.0.0.1:1355->127.0.0.1:1356: wsasend: An existing connection was forcibly closed by the remote host.
2019-03-26T12:26:36.3863679 [Info] [openvpn-process] Stderr: stream ended
2019-03-26T12:26:36.3883631 [Info] [openvpn-process] Stdout: stream ended
2019-03-26T12:26:36.4302519 [Trace] [session-storage] Session fd422116-09ca-4ab0-8d54-9c6674784bf5 updated
2019-03-26T12:26:36.4312489 [Debug] [NATS.Sender] Request '0xe5f5a25100ef313f95bf912a2e4fa65ba096dabd.0x4bf4ad3d7b54c4c014cecbe752b3d8ad187e202b.session-destroy' sending: {"payload":{"session_id":"fd422116-09ca-4ab0-8d54-9c6674784bf5"},"signature":"JPlQudDrVLu6SFQXYO/N6iqUHQkHQfG3zZDqKy5E0hhssLiGXzVCGzH2qok3Uwa+ZARz0FT30sY/wZNRVSeE0QE="}
2019-03-26T12:26:36.6207633 [Info] [Mysterium.api] Session stats sent: fd422116-09ca-4ab0-8d54-9c6674784bf5
2019-03-26T12:26:36.6207633 [Debug] [session-stats-sender] Final stats sent
2019-03-26T12:26:39.482932 [Info] [Mysterium.api] Total proposals: 20 supported: 20
2019-03-26T12:26:42.5771034 [Info] [ipify.api] IP detected: 88.119.191.24
2019-03-26T12:26:42.5880708 [Info] [ipify.api] IP detected: 88.119.191.24
2019-03-26T12:26:44.4848856 [Info] [Mysterium.api] Total proposals: 20 supported: 20
2019-03-26T12:26:46.4313728 [Info] [session.DestroyProducer] Session destroy request failed: "failed to send request '0xe5f5a25100ef313f95bf912a2e4fa65ba096dabd.0x4bf4ad3d7b54c4c014cecbe752b3d8ad187e202b.session-destroy'. nats: timeout"
2019-03-26T12:26:46.4313728 [Warn] [connection-manager] cleanup error:failed to send request '0xe5f5a25100ef313f95bf912a2e4fa65ba096dabd.0x4bf4ad3d7b54c4c014cecbe752b3d8ad187e202b.session-destroy'. nats: timeout
```

**Comment by zolia** [comment]
reset with master

**Comment by vkuznecovas** [comment]
Confirming that this now actually works properly with the latest master. Fixed by #815
